client/src/Login/__tests__/Login.test.js

```diff
@@ -36,14 +35,6 @@ describe('<Login />', () => {
       wrapper = shallow(<Login />)
     })
 
-    it('calls ReactGA.pageview()', () => {
```

Comment:
@katelovescode We could add these tests back but we have to enable ReactGA on `test` mode as well. Let me know what you think.

Reply:
I think it's fine to remove them.